docs/Advanced/ELK.stories.mdx

```diff
@@ -0,0 +1,224 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Docs/Advanced/Eclipse Layout Kernel" />
```

Comment:
Let's move this under `Getting Started` as `Layout` and then go into the whole ELK thing.